File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/felles/Næring.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.felles;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class Næring implements Virksomhetsklassifikasjon {
    private String kode;
    private String navn;
}

File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/felles/Næring.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.felles

data class Næring(
    private val kode: String,
    private val navn: String
) : Virksomhetsklassifikasjon {
    override fun getNavn(): String {
        return navn
    }

    override fun getKode(): String {
        return kode
    }
}